site/styles/_slider.scss

```diff
@@ -93,15 +85,12 @@ s72-slider.has-next.s72-slider-scroll::after {
 }
 
 .slider {
-  // not sure why this is needed for sliders with only a couple of items, that dont require sliding.
   width: 100%;
 }
 
 .slider-fluid {
   position: relative;
-  // This changes the default value for s72-slider due to us using the fade-in/fade-out to show more items
-  // We instead use .slider-fluid as the relative positioned parent so the fades dont move when scrolling.
-  // removing this value will break those fades.
```

Comment:
Aren't some of these comments still valuable?

Reply:
Maybe :/

Reply:
I mean, I'm not asserting they are, I don't know - just asking the question.